docs/guide/ingress/annotations.md

```diff
@@ -124,6 +125,16 @@ Traffic Listening can be controlled with following annotations:
         alb.ingress.kubernetes.io/ip-address-type: ipv4
         ```
 
+- <a name="customer-owned-ipv4-pool">`alb.ingress.kubernetes.io/customer-owned-ipv4-pool`</a> specifies the customer-owned IPv4 address pool for ALB on Outpost.
+    
+    !!!warning ""
+        This annotation should be treated as immutable. To remove or change coIPv4Pool, you need to recreate Ingress.
+
```

Comment:
How will be this surfaced to the user in the current approach if they still end up changing it?

Reply:
added a check for drifted settings.
Now customer will saw errors in controller log and Ingress events if they changed it.